src/Widgets/FadeBin.vala

```diff
@@ -0,0 +1,150 @@
+public class Tuba.Widgets.FadeBin : Gtk.Widget {
+	const int MAX_HEIGHT = 300;
+	const float FADE_HEIGHT = 125f;
+
+	private Gtk.Widget? _child = null;
```

Comment:
`unowned`

Reply:
Honestly, I'm scared of using unowned / weak ([`At the moment weak and unowned can be used interchangeably`](https://wiki.gnome.org/Projects/Vala/ReferenceHandling)) anymore. Every segfault Tuba has had is due to using them because, unless there's some secret way to do this that I don't know of, null checking is not enough and will happily pass them if it has been freed and then segfault.

Latest #1340 #1347 (also, both cases are very weird since the status should still hold a reference to them anyway!).

I guess it won't hurt testing it out and see if we hit it.